emisor/test/test_basic_meshlib_uninitialized.c

```diff
@@ -0,0 +1,29 @@
+#include "test_common.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <semaphore.h>
+
+#include "common/structures.c"
+
+#include "mesh_emitter.h"
+
+void * shm_ptr;
+int buffer_size;
+
+void test_when_shm_was_not_initialized_should_throw_error() {
+    printf("### Running test_when_shm_was_not_initialized_should_throw_error\n");
+    printf("Don't panic, next print should be error\n");
+    void *shm_ptr = mesh_register_emitter();
+    expect_equal(*((int *)shm_ptr), -1, "Should return -1 when shm was not initialized\n");
```

Comment:
mesh_register returns -1 when mesh hasn t been initialized by the initializer process